src/components/UserPost/UserPost.jsx

```diff
@@ -1,11 +1,91 @@
+import { useState } from "react"
+import {
+    AiFillLike
+} from "../../assets/react-icons"
 import './UserPost.css'
 
-function UserPost()
+function UserPost({userPostDetails})
 {
-    //This is temporary layout, will add user post content to it in future PRs
+    
+    const {
+        contentText,
+        userName,
+        userProfilePic,
+        noOfLikes
+    } = userPostDetails;
+
+    const [ showFullText, setShowFullText ] = useState(false)
+    let postText1 = "", postText2 = "";
+    
+    let postTextArray = contentText.split(" ")
+
+    if(postTextArray.length<45)
+    {
+        postText1 = contentText;
+    }
+    else
```

Comment:
Didn't understand what you are doing here

Reply:
Ooh, it is for Facebook like "...See More" functionality when post word limit exceeds 45